test/files/neg/t6567.check

```diff
@@ -1,9 +1,9 @@
-t6567.scala:10: warning: Suspicious application of an implicit view (Test.this.a2b) in the argument to Option.apply.
-  Option[B](a)
-           ^
-t6567.scala:12: warning: Suspicious application of an implicit view (Test.this.a2b) in the argument to Option.apply.
-  val b: Option[B] = Option(a)
-                           ^
+t6567.scala:10: warning: Suspicious application of an implicit view (Test.this.a2b) in the argument to Option.ofNullable.
+  Option.ofNullable[B](a)
+                      ^
+t6567.scala:12: warning: Suspicious application of an implicit view (Test.this.a2b) in the argument to Option.ofNullable.
+  val b: Option[B] = Option.ofNullable(a)
+                                      ^
```

Comment:
For some reason, changing from syntactic sugar of `apply` to `ofNullable` removes these warnings (I'm going to check what happens when explicitly calling `apply`). Anyone know why? cc @retronym